test/games/strategy/triplea/delegate/WW2V3_41_Test.java

```diff
@@ -1648,7 +1648,6 @@ public void testOccupiedTerrOfAttachmentWithCapital() {
       // Set as NEW capital
       taKiangsu.setCapital("Chinese");
     } catch (final GameParseException e) {
-      // TODO Auto-generated catch block
```

Comment:
Same, poor error handling. We lose the message contained in the game parse exception, and we'll have little info about why it happened.